### PR TITLE
Back to using core 12

### PR DESCRIPTION
**Pull request by PavelBal**
Back to 12..

### CODE REVIEW

File: packages.config [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-  <package id="OSPSuite.Infrastructure" version="11.2.155"/>
-  <package id="OSPSuite.Infrastructure.Import" version="11.2.155"/>
-  <package id="OSPSuite.Infrastructure.Autofac" version="11.2.155"/>
-  <package id="OSPSuite.R" version="11.2.155"/>
+  <package id="OSPSuite.Assets" version="12.0.218"/>
+  <package id="OSPSuite.Core" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure.Import" version="12.0.218"/>
+  <package id="OSPSuite.Infrastructure.Autofac" version="12.0.218"/>
+  <package id="OSPSuite.R" version="12.0.218"/>
```
Comment (Yuri05) [inline packages.config:8]: let's use the latest **12.0.226** 😉

File: dimensions
Comment (Yuri05) [inline dimensions:1]: why downgrading the dimensions version?

File: pkparameters
Comment (Yuri05) [inline pkparameters:1]: why downgrading the pk parameters version?

Comment (PavelBal) [reply pkparameters:1]: I actually do not understand this change... How do you even know it's a downgrade? The folders "pkparameters" and "dimensions" were marked as "changed" forever and now they slipped into this PR...

File: packages.config (hunk 1 above)
Comment (PavelBal) [reply packages.config:8]: Is it already used by PK-Sim?

File: pkparameters
Comment (Yuri05) [reply pkparameters:1]: > How do you even know it's a downgrade?

The latest commit in the dimensions repo is currently used by ospsuite-R.
Every change can be only a downgrade :)

Comment (PavelBal) [reply pkparameters:1]: Yes that did the trick.. I thought "Update submodules..." does exactly this, but apparently not...

File: packages.config (hunk 1 above)
Comment (Yuri05) [reply packages.config:8]: PK-Sim currently uses 12.0.223 
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/0b33736c8b7efe83c2faf743fd12a54587d3e4bb/src/PKSim.Core/PKSim.Core.csproj#L31-L39

Comment (PavelBal) [reply packages.config]: Updated to 223

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**